test/jest/LambertCoordinaat.test.js

```diff
@@ -0,0 +1,191 @@
+import LambertCoordinaat from '../../src/lambert-coordinaat';
+
+describe('Lambert Coördinaat', () => {
+  test('of - een vrije tekst', () => {
```

Comment:
ik zie ook altijd graag in de naam van de test wat "het verwachte gedrag" is ...

Reply:
klopt is wel beter